src/main/java/org/testng/xml/internal/XmlSuiteUtils.java

```diff
@@ -11,45 +11,10 @@
 import java.util.Set;
 
 /**
- * A utility class that exposes helper methods to work with {@link XmlSuite}
+ * A utility class to work with {@link XmlSuite}
  */
 public final class XmlSuiteUtils {
 
-    private XmlSuiteUtils() {
```

Comment:
Please re-store this private constructor, because this class is still a utility class which has only static methods.

Reply:
Done